resources/views/ItemList/Components/Filter/ItemColorTiles.twig

```diff
@@ -0,0 +1,12 @@
+<script type="x/template" id="vue-item-color-tiles">
+    <div class="fl-item-color-tiles-container">
+        <ul class="item-color-tiles-list">
+            <li class="item-color-tiles-list-item" v-for="colorValue in facet.values">
+                <label class="color-tile-label" @click="tileClicked(colorValue.name)">
+                    <div v-bind:style="{background: colorValue.hexValue }"></div>
+                    <img v-show="isSelected(colorValue.name)" src="{{ plugin_path('Findologic') }}/images/selectedColorTile.svg">
```

Comment:
Remove file `selectedColorTile.svg` from the project and use `https://cdn.findologic.com/login.symfony/web/autocomplete/img/selected.svg` as value for attribute `src`.